### PR TITLE
Prompt users to migrate to org billing if trying to enable branching but on legacy billing

### PR DESCRIPTION
**Pull request by joshenlim**
image

### CODE REVIEW

File: studio/components/layouts/AppLayout/EnableBranchingButton/EnableBranchingModal.tsx [hunk 1]
```diff
@@ -80,88 +90,129 @@ const EnableBranchingModal = () => {
...
+        {!isOrgBilling ? (
           <>
-            <Modal.Separator />
-            <Modal.Content className="px-7 py-6">
-              <GenericSkeletonLoader />
+            <Modal.Content className="px-3 border-y bg-alternative">
+              <Alert_Shadcn_ variant="default" className="rounded-none border-0">
+                <IconAlertCircle strokeWidth={2} />
+                <AlertTitle_Shadcn_>
+                  You will need to migrate to organization-based billing first
```
Comment (MildTomato) [inline studio/components/layouts/AppLayout/EnableBranchingButton/EnableBranchingModal.tsx:103]: need less technical language

```suggestion
                  Organization-based billing migration required
```

File: studio/components/layouts/AppLayout/EnableBranchingButton/EnableBranchingModal.tsx [hunk 2]
```diff
@@ -80,88 +90,129 @@ const EnableBranchingModal = () => {
...
+                </AlertTitle_Shadcn_>
+                <AlertDescription_Shadcn_>
+                  <p className="!leading-normal">
+                    Your project is currently under the legacy billing system and we require
+                    projects to migrate to organization-based billing prior to enabling database
+                    branching for their projects.
```
Comment (MildTomato) [inline studio/components/layouts/AppLayout/EnableBranchingButton/EnableBranchingModal.tsx:109]:
```suggestion
                    Your organization will first need to be migrated to use our new organization-based billing before you can enable Branching.
```

**Comment by MildTomato** [comment]
@joshenlim Can we just have the CTA go immediately to the migration?

And we can't use "Head to..." in text i don't think, would just be "Go to.." or just state "Organization Settings" wdyt ?

**Comment by joshenlim** [comment]
>re. the current CTA text:
And we can't use "Head to..." in text i don't think, would just be "Go to.." or just state "Organization Settings" wdyt ?

yeah i think that should be okay! I think just "Organization Settings' will be good then

File: studio/components/layouts/AppLayout/EnableBranchingButton/EnableBranchingModal.tsx [hunk 3]
```diff
@@ -80,88 +90,127 @@ const EnableBranchingModal = () => {
...
+                <Button
+                  disabled={isCreating}
+                  type="default"
+                  onClick={() => snap.setShowEnableBranchingModal(false)}
+                >
+                  Understood
+                </Button>
+                <Link passHref href={`/org/${selectedOrg?.slug}/billing`}>
+                  <Button asChild>
+                    <a>Head to organization settings</a>
```
Comment (joshenlim) [inline studio/components/layouts/AppLayout/EnableBranchingButton/EnableBranchingModal.tsx:126]:
```suggestion
                    <a>Organization settings</a>
```

**Review by MildTomato — APPROVED** [review]

**Pull request merged**